exporters/stdout/stdoutmetric/example_test.go

```diff
@@ -33,12 +34,10 @@ const (
 )
 
 var (
-	// TODO Bring back Global package
-	// meter = global.GetMeterProvider().Meter(
-	// 	instrumentationName,
-	// 	metric.WithInstrumentationVersion(instrumentationVersion),
-	// )
-	meter metric.Meter
+	meter = global.Meter(
+		instrumentationName,
+		metric.WithInstrumentationVersion(instrumentationVersion),
+	)
```

Comment:
We only need to do this in one place, either here or on line 86.

Reply:
Great catch! Done!